src/network/serverpackethandler.cpp

```diff
@@ -1127,7 +1127,7 @@ void Server::handleCommand_Damage(NetworkPacket* pkt)
 		return;
 	}
 
-	if (g_settings->getBool("enable_damage")) {
+	if (g_settings->getBool("enable_damage") && !playersao->getDeadStatus()) {
```

Comment:
Couldn't this be done without the new variable and just adding the check `&& playersao->getHP() > 0` here?